nequip/nn/_graph_mixin.py

```diff
@@ -78,6 +78,25 @@ def _init_irreps(
         new_out.update(irreps_out)
         self.irreps_out = new_out
 
+    def add_independent_irreps(self, irreps: Dict[str, Any]):
```

Comment:
```suggestion
    def _add_independent_irreps(self, irreps: Dict[str, Any]):
```

this is too weird to be public :smile: